### PR TITLE
 Adds handlebar layout and partials

### PR DESCRIPTION
**Pull request by Jen-Harris**
- Basic structure of form in place 
- Still need to finish off some of the questions

### CODE REVIEW

File: src/views/about.hbs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{{> header }}
+<section>
```
Comment (dangerdak) [inline src/views/about.hbs]: should this be a form? :angel:

File: src/views/background.hbs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{{> header }}
+<section>
```
Comment (dangerdak) [inline src/views/background.hbs]: should this also be a form? :angel:

File: src/views/symptoms.hbs [hunk 3]
```diff
@@ -0,0 +1,47 @@
+{{> header }}
+
+<section>
```
Comment (dangerdak) [inline src/views/symptoms.hbs]: formy formy

**Comment by Jen-Harris** [comment]
@dangerdak @polyccon forms now in place 👍

**Pull request merged**